R/qenv-extract.R

```diff
@@ -0,0 +1,48 @@
+#'
+#' @section Subsetting:
+#' `x[names]` subsets objects in `qenv` environment and limit the code to the necessary needed to build limited objects.
+#'
+#' @param names (`character`) names of objects included in `qenv` to subset
```

Comment:
please mention about warnings

Reply:
Good idea! Will mention that some objects are skipped if not present, but the warning is raised!

Reply:
6338ce13a034595c19bac9c305cf607d00d65073